pylint/checkers/dunder_methods.py

```diff
@@ -0,0 +1,148 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+from typing import TYPE_CHECKING
+
+from astroid import nodes
+
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+
+if TYPE_CHECKING:
+    from pylint.lint import PyLinter
+
+
+class DunderCallChecker(BaseChecker):
+    """Check for unnecessary dunder method calls."""
+
+    __implements__ = IAstroidChecker
+
+    # Docs: https://docs.python.org/3/reference/datamodel.html#basic-customization
```

Comment:
Is there a reason this was reverted from being in the docstring ? I don't mind, but just to make sure other changes were not removed too :)

Reply:
whoops my bad, messed up when rebasing after the 2.13 release and forgot to add this back in. I've moved it back to the docstring and the other changes are still fine 😄